apps/samourai-server/docker-compose.yml

```diff
@@ -104,6 +104,7 @@ services:
         WHIRLPOOL_RESYNC: "on"
         WHIRLPOOL_DEBUG: "off"
         WHIRLPOOL_DEBUG_CLIENT: "off"
+        WHIRLPOOL_COORDINATOR_ONION: "on"
```

Comment:
Do we need this on since we run our own Tor hidden service proxy?

Reply:
> When set to on, whirlpool-cli will contact the coordinator through its onion address. When set to off, clearnet address will be used (through Tor).

I think it is better to use the coordinator onion address, isn't it?

Reply:
Ah perfect, yes indeed.